packages/client/vite.config.ts

```diff
@@ -70,51 +70,48 @@ function targetBuildPlugin(dev: boolean, target: string): Plugin {
         const mockPath = `${file.dir}/${file.name}.${target}.${extension}`
         return this.resolve(mockPath, importer)
       } else {
-        const rootPrefix = "src/client/"
-        const thisImporter = (importer || "").replace(/\\/g, "/")
         if (
           !importer ||
-          !thisImporter.includes(rootPrefix) ||
+          importer.includes("node_modules") ||
           source === "./main"
         ) {
           return null
         }
 
-        const importedFile = path.parse(source)
+        if (!source.startsWith(".") && !source.startsWith("/")) {
```

Comment:
our imports should be relative (./ or ../) or aliased (/src/...) .. ignoring `~/` as that is an anomaly used once in storybook code that should not really be there, but SB stuff going away with introduction of UISK I hope